### PR TITLE
closes [remove human judgement ensemble model #41]

### PR DESCRIPTION
**Pull request by matthewcornell**

### CODE REVIEW

File: src/predtimechart.js [hunk 1]
```diff
@@ -1021,20 +798,6 @@ const App = {
...
-                        this.state.forecasts[USER_ENSEMBLE_MODEL.name] = _calcUemForecasts(USER_ENSEMBLE_MODEL.models, this.state.forecasts);  // replaces if present
-                        USER_ENSEMBLE_MODEL.lastError = null;
-                        console.debug('fetchDataUpdatePlot(): forecasts:', this.state.forecasts[USER_ENSEMBLE_MODEL.name]);
-                    } catch (error) {
-                        USER_ENSEMBLE_MODEL.lastError = error;
-                        console.error(`fetchDataUpdatePlot(): error calling _calcUemForecasts(): ${error}`);
-                    }
-                }
-
-                this.updateModelsList();
```
Comment (elray1) [inline src/predtimechart.js:1037]: double checking on the deletion of `this.updateModelsList();`.  Seems like we might still want that?  I'm not confident.

**Review by elray1 — COMMENTED** [review]
Looks good overall.  There was one deletion where I was unsure about whether we might want to keep a line.  And I am wondering if we also want to remove these lines? https://github.com/reichlab/predtimechart/pull/47/files#diff-de5803944c532bb50f10888d5b3f3a3c649d9c398ab5fc94220f864baa322de4R504-R549

**Comment by matthewcornell** [comment]
Two great catches, Evan! I was unsure about that line myself, and my re-review should that it's necessary. The block of lines is now unused, so I deleted that function and the call to it. I think we're good now.

**Review by elray1 — APPROVED** [review]
lgtm; thanks!

**Pull request merged**